### DISCUSSION TITLE
Popup size change

### DISCUSSION THREAD

**Discussion by mAmineChniti** [folke/noice.nvim#750] [Q&A] [answered]
Hey, i was wondering how can i change the popup box's width so it wouldn't take a large part of the screen, if i had multiple popups like this they'd just fully cover the editor
![WindowsTerminal_opQ95XDJUC](https://github.com/folke/noice.nvim/assets/124379229/c65aada1-0a60-418c-aeb4-f2e2abb93830)
This is my current noice.nvim config
```lua
{
    'folke/noice.nvim',
    event = 'VeryLazy',
    opts = {},
    dependencies = {
      'MunifTanjim/nui.nvim',
      'rcarriga/nvim-notify',
    },
    config = function()
      require('noice').setup {
        lsp = {
          override = {
            ['vim.lsp.util.convert_input_to_markdown_lines'] = true,
            ['vim.lsp.util.stylize_markdown'] = true,
            ['cmp.entry.get_documentation'] = true,
          },
        },
        presets = {
          bottom_search = true,
          command_palette = true,
          long_message_to_split = true,
          inc_rename = false,
          lsp_doc_border = false,
        },
      }
    end,
  },
```

**Comment by ilias777** [comment, marked as answer]
Try to set the `max_width` value in the `nvim-notify` options.

Look here &rarr; [Notify docs](https://github.com/rcarriga/nvim-notify/blob/d333b6f167900f6d9d42a59005d82919830626bf/doc/nvim-notify.txt#L189)

**Reply by mAmineChniti** [reply]
Thank you
